Modules/faulthandler.c

```diff
@@ -20,6 +20,19 @@
 #  include <sys/resource.h>
 #endif
 
+/* Using an alternative stack requires sigaltstack()
+   and sigaction() SA_ONSTACK */
+#if defined(HAVE_SIGALTSTACK) && defined(HAVE_SIGACTION)
+#  define FAULTHANDLER_USE_ALT_STACK
+#endif
+
+#ifdef FAULTHANDLER_USE_ALT_STACK
+#ifdef HAVE_LINUX_AUXVEC_H
+#  include <linux/auxvec.h>
+#endif
+#include <sys/auxv.h>
+#endif
```

Comment:
It's unclear to me why <sys/auxv.h> is always included, but <linux/auxvec.h> is included conditionally. Would it be possible that AT_MINSIGSTKSZ is defined by <sys/auxv.h>?

```suggestion
#ifdef FAULTHANDLER_USE_ALT_STACK
#  ifdef HAVE_LINUX_AUXVEC_H
#    include <linux/auxvec.h>
#  endif
#  include <sys/auxv.h>
#endif
```

Reply:
`linux/auxvec.h` does not exist on OSX, but without it `getauxval` (include from `auxv.h`) is not used, so I will move the `#include <sys/auxv.h>` inside the `HAVE_LINUX_AUXVEC_H` conditional.

Reply:
Changes applied.